### PR TITLE
reorganizing directory structure and adding readme

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Per the discussion on #286 , I'm reorganizing the directory and adding a README.

### CODE REVIEW

File: scripts/us_census/fred/homeownership/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Scripts for importing datasets from FRED data on Kaggle
+
+This directory stores all scripts used to import datasets into Data Commons from FRED (Federal Reserve Economic Data, from the St. Louis Fed) via the
+US Census Bureau's account on Kaggle.
```
Comment (tjann) [inline scripts/us_census/fred/homeownership/README.md:4]: Can you add some links. Also here is an example README (feel free to make suggestions to this template): https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset

**Review by tjann — CHANGES_REQUESTED** [review]
I think this is ok, just not too sure what the subdirectory should be. FRED? Kaggle? The actual original data source from Census (e.g. Population Estimates Program, American Housing Survey, etc.) -- I can make a better call with the link to the kaggle page.

**Comment by KilimAnnejaro** [comment]
@tjann the Kaggle link is https://www.kaggle.com/census/homeownership-rate-time-series-collection.

**Comment by tjann** [comment]
> @tjann the Kaggle link is https://www.kaggle.com/census/homeownership-rate-time-series-collection.

Thanks. In that case I think the source is https://www.census.gov/housing/hvs/index.html and we can do `us_census/cps_hvs/`

File: scripts/us_census/cps_hvs/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- Documentation: <https://www.kaggle.com/census/homeownership-rate-time-series-collection>
+- Data Visualization UI: <https://fred.stlouisfed.org/series/MNHOWN>
+
+## About the import
+
+### Artifacts
+
+#### Raw Data
+- <https://github.com/datacommonsorg/data/blob/master/scripts/us_census/cps_hvs/MNHOWN.csv>: Historical homeownership rate in Minnesota.
```
Comment (KilimAnnejaro) [inline scripts/us_census/cps_hvs/README.md:36]: This link does not work at this time, but it should after the PR is merged.

File: scripts/us_census/cps_hvs/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Template MCF
+- <https://github.com/datacommonsorg/data/blob/master/scripts/us_census/cps_hvs/Homeownership_Rate.tmcf>: TMCF for homeownership rate data.
```
Comment (KilimAnnejaro) [inline scripts/us_census/cps_hvs/README.md:39]: As above.

File: scripts/us_census/fred/homeownership/README.md (hunk 1 above)
Comment (KilimAnnejaro) [reply scripts/us_census/fred/homeownership/README.md]: Hi @tjann, thanks for the example README! I followed it pretty closely and anticipate linking to it in future documentation.

In the process of adding documentation, I realized that the CSV contained two columns that were not used in the final knowledge graph, so I went ahead and removed them. I know that changes this PR's scope a bit--if you'd like me to move those changes to another PR, let me know!

Comment (tjann) [reply scripts/us_census/fred/homeownership/README.md]: @KilimAnnejaro thanks! Feel free to improve upon it too!

Yeah, could you revert that change? It's totally ok for the CSV to have more columns than used. There's no need to remove them, and is a selling point here (less pre-processing).

Comment (KilimAnnejaro) [reply scripts/us_census/fred/homeownership/README.md]: @tjann reverted!

File: scripts/us_census/cps_hvs/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+### Import Procedure
+
+To import new homeownership data from FRED into Data Commons, simply upload the CSVs from Kaggle, removing the _realtime_start_ and _realtime_end_ columns manually.
```
Comment (tjann) [inline scripts/us_census/cps_hvs/README.md:43]: I think the latter half of the sentence can be removed.

**Review by tjann — APPROVED** [review]

File: scripts/us_census/cps_hvs/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## About the dataset
+
+### Download URL
+
+The CSV file is available for download from <https://www.kaggle.com/census/homeownership-rate-time-series-collection>.
+
+### Overview
+
+This dataset contains two variables:
+1. date: The date when the value was measured.
```
Comment (tjann) [inline scripts/us_census/cps_hvs/README.md:19]: I don't think date is considered a variable, I would say there is just one variable here

**Review by tjann — APPROVED** [review]

**Pull request merged**